### ISSUE TITLE
Delete the __main__ in twisted/protocols/postfix.py

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#3717]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)| @exarkun reported|
|-|-|
|Trac ID|trac#3717|
|Type|enhancement|
|Created|2009-03-30 19:28:29Z|
|Branch|https://github.com/twisted/twisted/tree/postfix-main-3717|

If there is supposed to be a runnable script, it should be a separate, real thing.  With tests.  Etc.

Attachments:

* [postfix.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/c46/c464a6a0a28073fb8e4a26db2da76d13c53c0dea/a8c0de9a474c8765391e7bbfe3c9afd794973e23.patch) (747 bytes) - added by cary on 2009-03-30 21:40:56Z - 

<details><summary>Searchable metadata</summary>

```
trac-id__3717 3717
type__enhancement enhancement
reporter__exarkun exarkun
priority__normal normal
milestone__ 
branch__branches_postfix_main_3717 branches/postfix-main-3717
branch_author__thijs__cary thijs, cary
status__closed closed
resolution__fixed fixed
component__core core
keywords__ 
time__1238441309000000 1238441309000000
changetime__1238511740000000 1238511740000000
version__None None
owner__ 
cc__thijs
```
</details>

**Comment by twisted-trac** [comment]
|<img alt="cary's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|cary removed owner|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_2"></a>|@thijstriemstra commented|
|-|-|

(In [26546]) Branching to 'postfix-main-3717'

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_3"></a>|@thijstriemstra set owner to @thijstriemstra<br>@thijstriemstra set status to `assigned`|
|-|-|

Replying to [cary](#note_1):

What about moving it into something like [twisted/doc/core/examples](twisted/doc/core/examples)/postfix.py instead?

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_4"></a>|@thijstriemstra commented|
|-|-|

(In [26548]) Add postfix sample refs [#3717](https://github.com/twisted/twisted/issues/3717)

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)|@thijstriemstra removed owner<br>@thijstriemstra set status to `new`|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_6"></a>|@exarkun set owner to @exarkun|
|-|-|

Looks pretty good.  Some minor coding standard issues should be addressed in postfix.py.  I'm going to do that now.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_7"></a>|@exarkun commented|
|-|-|

(In [26568]) Update copyright date, module docstring formatting, and import ordering

refs [#3717](https://github.com/twisted/twisted/issues/3717)

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_8"></a>|@exarkun set status to `closed`|
|-|-|

(In [26573]) Merge postfix-main-3717 

Author: cary, thijs
Reviewer: exarkun
Fixes: [#3717](https://github.com/twisted/twisted/issues/3717)

Move the `__main__` out of `twisted/protocols/postfix.py` into a separate example
script.

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
